.github/ISSUE_TEMPLATE/bug_report_form.yaml

```diff
@@ -43,7 +44,7 @@ body:
     attributes:
       label: System information
       description: Please complete the following information
-      value: "- Database version: [e.g. Mysql 5.7, SQLite]\n- PHP version: [e.g. PHP 8.1]\n- Installation type: [e.g. Docker, Docker image source, git, Yunohost]\n-Web server type: [e.g. Apache, nginx]\n- Device: [e.g. iPhone13]\n- OS: [e.g. Ubuntu 22.04, Win10, MacOS14]\n- Browser: [e.g. Firefox 124]"
+      value: "- Database version: [e.g. Mysql 5.7, SQLite]\n- PHP version: [e.g. PHP 8.1]\n- Installation type: [e.g. Docker, Docker image source, git, Yunohost]\n- Web server type: [e.g. Apache, nginx]\n- Device: [e.g. iPhone13]\n- OS: [e.g. Ubuntu 22.04, Win10, MacOS14]\n- Browser: [e.g. Firefox 124]"
```

Comment:
```suggestion
      value: "- Database version: [e.g. SQLite, PostgreSQL 17]\n- PHP version: [e.g. PHP 8.2]\n- Installation type: [e.g. Docker, Docker image source, git, Yunohost]\n- Web server type: [e.g. Apache, nginx]\n- Device: [e.g. iPhone13]\n- OS: [e.g. Ubuntu 24.04, Win11, MacOS14]\n- Browser: [e.g. Firefox 134]"
```

A few updates

Reply:
Any reasons why MySQL is not anymore on the list?

Reply:
They're just examples, I don't think it matters much one way or the other? (But shouldn't it say SQLite 3.48.0)

Reply:
(@Frenzie was faster :-P) Regarding the version of SQLite, as you prefer. It might be difficult for users to find out (besides via our own *About* system) and is in most cases dependent to the PHP version.

Reply:
When the current `edge` is published as 1.26.0 I will add the information where to find the system environment information for finding the information fast

Reply:
Let's try to publish 1.26.0 very soon